TramsDataApi.Test/UseCases/GetEstablishmentByUkprnTests.cs

```diff
@@ -12,648 +14,94 @@ namespace TramsDataApi.Test.UseCases
-                    PersonalDevelopment = misEstablishment.PersonalDevelopment.ToString(),
-                    EffectivenessOfLeadershipAndManagement =
-                        misEstablishment.EffectivenessOfLeadershipAndManagement.ToString(),
-                    SafeguardingIsEffective = misEstablishment.SafeguardingIsEffective,
-                    EarlyYearsProvision = misEstablishment.EarlyYearsProvisionWhereApplicable.ToString(),
-                    SixthFormProvision = misEstablishment.SixthFormProvisionWhereApplicable.ToString(),
-                    PreviousFullInspectionNumber = misEstablishment.PreviousFullInspectionNumber,
-                    PreviousInspectionStartDate = misEstablishment.PreviousInspectionStartDate,
-                    PreviousInspectionEndDate = misEstablishment.PreviousInspectionEndDate,
-                    PreviousPublicationDate = misEstablishment.PreviousPublicationDate,
-                    PreviousFullInspectionRelatesToUrnOfCurrentSchool =
-                        misEstablishment.DoesThePreviousFullInspectionRelateToTheUrnOfTheCurrentSchool,
-                    UrnAtTheTimeOfPreviousFullInspection = misEstablishment.UrnAtTimeOfPreviousFullInspection.ToString(),
-                    LAESTABAtTheTimeOfPreviousFullInspection =
-                        misEstablishment.LaestabAtTimeOfPreviousFullInspection.ToString(),
-                    SchoolNameAtTheTimeOfPreviousFullInspection =
-                        misEstablishment.SchoolNameAtTimeOfPreviousFullInspection,
-                    SchoolTypeAtTheTimeOfPreviousFullInspection =
-                        misEstablishment.SchoolTypeAtTimeOfPreviousFullInspection,
-                    PreviousFullInspectionOverallEffectiveness =
-                        misEstablishment.PreviousFullInspectionOverallEffectiveness,
-                    PreviousCategoryOfConcern = misEstablishment.PreviousCategoryOfConcern,
-                    PreviousQualityOfEducation = misEstablishment.PreviousQualityOfEducation.ToString(),
-                    PreviousBehaviourAndAttitudes = misEstablishment.PreviousBehaviourAndAttitudes.ToString(),
-                    PreviousPersonalDevelopment = misEstablishment.PreviousPersonalDevelopment.ToString(),
-                    PreviousEffectivenessOfLeadershipAndManagement =
-                        misEstablishment.PreviousEffectivenessOfLeadershipAndManagement.ToString(),
-                    PreviousIsSafeguardingEffective = misEstablishment.PreviousSafeguardingIsEffective,
-                    PreviousEarlyYearsProvision = misEstablishment.PreviousEarlyYearsProvisionWhereApplicable.ToString(),
-                    PreviousSixthFormProvision = misEstablishment.PreviousSixthFormProvisionWhereApplicable
-                },
-                MISFurtherEducationEstablishment = null,
-                SMARTData = null,
-                Financial = null,
-                Concerns = null
-            };
```

Comment:
100% agree with moving this stuff out of `GetEstablishmentByUkprnTests`, but this then means that our factories have no unit tests around them, as they were tested through this unit. Could you move these into a test for `EstablishmentResponseFactory` instead?